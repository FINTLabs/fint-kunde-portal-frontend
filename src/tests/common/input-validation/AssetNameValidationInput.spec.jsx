import React from "react";
import { create } from "react-test-renderer";
import AssetNameValidationInput from "../../../common/input-validation/AssetNameValidationInput";
describe("Feature component", () => {
  const component = create(
    <AssetNameValidationInput
      name="test"
      className="test"
      title="test"
      assetNameIsValid={() => {}}
      onChange={() => {}}
    />
  );
  it("should match snapshot", () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
