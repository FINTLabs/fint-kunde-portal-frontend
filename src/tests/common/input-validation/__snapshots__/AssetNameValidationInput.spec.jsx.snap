// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature component should match snapshot 1`] = `
<div
  className="MuiFormControl-root test MuiFormControl-fullWidth"
>
  <label
    className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
    data-shrink={false}
    htmlFor="test"
  >
    test
    <span
      className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
    >
      â€‰
      *
    </span>
  </label>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={true}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      name="test"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={true}
      type="text"
    />
  </div>
  <p
    className="MuiFormHelperText-root Mui-required"
  >
    
  </p>
</div>
`;
